ci-operator/step-registry/ipi/conf/vsphere/check/ipi-conf-vsphere-check-commands.sh

```diff
@@ -116,12 +115,8 @@ source "${SHARED_DIR}/govc.sh"
 
 DATACENTERS=("$GOVC_DATACENTER")
 # If testing a zonal install, there are multiple datacenters that will need to be cleaned up
-# vlanid between 1287 and 1302 will use profile:vsphere-multizone-2
-if [ ${vlanid} -ge 1287 ] && [ ${vlanid} -le 1302 ]; then
-  DATACENTERS=(
-    "IBMCloud"
-    "datacenter-2"
-  )
+if [[ $JOB_NAME_SAFE == *"zones"* ]]; then
```

Comment:
The same comment as https://github.com/openshift/release/pull/46163#discussion_r1413309957